server/lib/configuration.js

```diff
@@ -451,6 +451,13 @@ const conf = module.exports = convict({
     }
   },
   sms: {
+    redirect: {
+      targetLink: {
+        default: 'https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=android&creative=button',
```

Comment:
@davismtl - This is the link specified in the Adjust tracker hierarchy, but having `android` embedded doesn't seem correct to me.